docs/defects-api.yml

```diff
@@ -30,7 +30,7 @@ paths:
           default:
```

Comment:
Can you rebase your branch against 9571 please, as this PR seems to be missing or removing existing code.